model.py

```diff
@@ -1957,6 +1957,13 @@ def lookup(cls, _db, name=None, viaf=None, lc=None, aliases=None,
 
         return contributors, new
 
+    @property
```

Comment:
These changes are good for the final version, but comment them out when you update the other components to make sure you find as much as possible of the code that needs to be changed.